BCStudentSoftwareDevTeam/celts
# Create student search page issue#4

Summary:
This PR resolves issue #4 
* Created a simple search bar on the student search page that the admin can use to search users by their username/first name/ last name. We also made sure that if the volunteer's name pops up and the admin clicks on their name, it will redirect them to the volunteer's profile page.


To test:
You can visit the web page with  URL: http://:8080/search_student.  To test, try search any user from the database either by their username/first name/ last name. Click their name and it should redirect you to their profile page.
